src/pages/mint/index.js

```diff
@@ -358,6 +358,14 @@ export const Mint = () => {
 
     const uri0 = `ipfs://${hashv0}`
     const uri1 = `ipfs://${hashv1}`
+
+    // Ignore IPFS URI's that are in the ignore list; they can be minted multiple times
+    const ignoreUriList = getIgnoreUriList()
+    if (ignoreUriList.includes(uri0) || ignoreUriList.includes(uri1)) {
+      console.log('ignore uri')
```

Comment:
remaining console logs :)

Btw how is this list generated?

Reply:
Not exactly sure what you mean by that.

Reply:
IMO there should be 0 log on a website, console.debug is meant for that if really needed.
After the refacto I wanted to convert every logs on the site (removing the useless ones)

Reply:
Will remove.